packages/yourloops/test/integration/assert/join-team.ts

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2023, Diabeloop
+ *
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import { screen, within } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+
+export const closeDialogJoinTeam = async () => {
+  const badgeTeamMenu = screen.getByLabelText('Open team menu')
+  await userEvent.click(badgeTeamMenu)
+  const buttonJoinTeam = screen.getByTestId('team-menu-teams-link')
+  await userEvent.click(buttonJoinTeam)
+  const dialog = screen.getByRole('dialog')
+  await userEvent.click(buttonJoinTeam)
+  const cancelButton = within(dialog).getByRole('button', { name: 'Cancel' })
+  await userEvent.click(cancelButton)
+  expect(screen.queryByRole('dialog')).not.toBeInTheDocument()
+}
+
+export const closeDialogPrivacyPolicy = async () => {
+  const badgeTeamMenu = screen.getByLabelText('Open team menu')
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**🤓 nitpick:** ‏All the method of this file are executing this line of code at first. I suggest having the `badgeTeamMenu` as a method parameter